tests/app/va/vetext/test_client.py

```diff
@@ -1,17 +1,18 @@
-import pytest
-import requests
 from base64 import b64encode
-from requests_mock import ANY
 from unittest import mock
 from uuid import uuid4
+
+import pytest
+from requests import HTTPError, Response
+from requests.exceptions import ConnectTimeout, ReadTimeout, RequestException
+from requests_mock import ANY
+
+from app.celery.exceptions import NonRetryableException, RetryableException
+from app.mobile_app import DEAFULT_MOBILE_APP_TYPE
```

Comment:
see comment above about fixing this longstanding typo `DEAFULT_MOBILE_APP_TYPE`

Reply:
Updated.